crates/swc_ecma_transforms_react/tests/jsx/fixture/runtime/jsxdev-args-with-fragment/output.js

```diff
@@ -0,0 +1,13 @@
+import { jsxDEV as _jsxDEV, Fragment as _Fragment } from "react/jsx-dev-runtime";
+var x = _jsxDEV(_Fragment, {
+    children: [
+        _jsxDEV("div", {
+            children: "hoge"
+        }, void 0, false, void 0, void 0),
+        _jsxDEV("div", {
+            children: "fuga"
+        }, void 0, false, void 0, void 0)
+    ]
+},
+void 0,
+true);
```

Comment:
I got different output from babel.

```js
var _jsxFileName = "/Users/kdy1/projects/lab/src/input.js";
import { jsxDEV as _jsxDEV } from "react/jsx-dev-runtime";
import { Fragment as _Fragment } from "react/jsx-dev-runtime";

var x = /*#__PURE__*/_jsxDEV(_Fragment, {
  children: [/*#__PURE__*/_jsxDEV("div", {
    children: "hoge"
  }, void 0, false, {
    fileName: _jsxFileName,
    lineNumber: 3,
    columnNumber: 9
  }, this), /*#__PURE__*/_jsxDEV("div", {
    children: "fuga"
  }, void 0, false, {
    fileName: _jsxFileName,
    lineNumber: 4,
    columnNumber: 9
  }, this)]
}, void 0, true);
```

Seems like children of fragments are not visited?

Reply:
Can you add `e.visit_mut_children_with(self)` to https://github.com/swc-project/swc/blob/79f6e2e2a406c6d7c98ea91884dd61e756b7925d/crates/swc_ecma_transforms_react/src/jsx_src/mod.rs#L38 and https://github.com/swc-project/swc/blob/79f6e2e2a406c6d7c98ea91884dd61e756b7925d/crates/swc_ecma_transforms_react/src/jsx_self/mod.rs#L39

Reply:
I think this works. `jsxdev-args-with-gragment` test is placed in `fixture`, not in `integration`. This means `jsx-src` and `jsx-self` does not work while this test run. This output will be the same with babel’s output if we move this test into `integration`. I think this is a little confusing, so I moved this test case from `fixture` into `integration` in fdaaaa2